lib/index.js

```diff
@@ -9,7 +9,13 @@ export default class Roots {
   compile () {
     return new Promise((resolve, reject) => {
       webpack(this.config, (err, stats) => {
+        // handle all errors
+        // ref: https://webpack.github.io/docs/node.js-api.html#error-handling
         if (err) { return reject(err) }
+        let jsonStats = stats.toJson()
+        if (jsonStats.errors.length > 0) { reject(jsonStats.errors) }
+        if (jsonStats.warnings.length > 0) { reject(jsonStats.warnings) } // we may want to WARN instead of reject
```

Comment:
@jenius we'll also need to decide how we want to handle a warning. i guess in an ideal world we'd just pass this warning on